index.html

```diff
@@ -1000,18 +1003,18 @@ <h3>Contexts</h3>
         </pre>
 
         <p>
-The example above uses the base context URI
+The example above uses the base context <a>URI</a>
 (<code>https://www.w3.org/2018/credentials/v1</code>) to establish that the
-conversation is about a <a>verifiable credential</a>. The second URI
-(<code>https://www.w3.org/2018/credentials/examples/v1</code>) establishes that the
-conversation is about examples.
+conversation is about a <a>verifiable credential</a>. The second <a>URI</a>
+(<code>https://www.w3.org/2018/credentials/examples/v1</code>) establishes that
+the conversation is about examples.
         </p>
 
         <p class="note">
-This document uses the examples context URI
-(<code>https://www.w3.org/2018/credentials/examples/v1</code>) for the purposes of
-demonstrating examples. The examples context URI must not be used for any
-other purpose, such as in pilot or production systems.
+This document uses the examples context <a>URI</a>
+(<code>https://www.w3.org/2018/credentials/examples/v1</code>) for the purposes
+of demonstrating examples. The examples context <a>URI</a> MUST NOT be used for
```

Comment:
This transforms a non-normative "must not" to a normative "MUST NOT" and would kick us into another Candidate Recommendation phase. Keep the non-normative nature of this statement, please.

Reply:
Fixed.